packages/main/src/components/SplitterLayout/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { SplitterElementPropTypes } from '@ui5/webcomponents-react/dist/SplitterElement';
+import { SplitterLayoutContext } from '@ui5/webcomponents-react/dist/SplitterLayoutContext';
+import { CommonProps } from '@ui5/webcomponents-react/interfaces/CommonProps';
+import React, { CSSProperties, forwardRef, ReactElement, Ref } from 'react';
+import { createUseStyles } from 'react-jss';
+import { styles } from './SplitterLayout.jss';
+import '@ui5/webcomponents-icons/dist/vertical-grip.js';
+import '@ui5/webcomponents-icons/dist/horizontal-grip.js';
+import { useConcatSplitterElements } from './useConcatSplitterElements';
+import clsx from 'clsx';
+
+const useStyles = createUseStyles(styles, { name: 'SplitterLayout' });
+
+export interface SplitterLayoutPropTypes extends CommonProps {
+  /**
+   * Controls if a vertical or horizontal `SplitterLayout` is rendered.
+   */
+  vertical?: boolean;
```

Comment:
This prop behaves incorrect in my opinion:
![image](https://user-images.githubusercontent.com/8775211/152297759-8321e204-3397-4d6f-8182-b2808c1fef8d.png)

This should be `vertical=true`, not `false`.

Reply:
Yes, you are right. I compared our implementation with the one of [SAPUI5](https://sapui5.hana.ondemand.com/#/entity/sap.ui.layout.Splitter/sample/sap.ui.layout.sample.Splitter) and there it's implemented the other way around. (The default for `orientation` is `"Horizontal"` so the splitter elements in the demo are initially horizontally aligned)